### DISCUSSION TITLE
Delete videos

### DISCUSSION THREAD

**Discussion by Al3eRtO** [roleoroleo/yi-hack-Allwinner-v2#582] [General]
Hi, Can I delete all videos at once from the web interface?

**Comment by roleoroleo** [comment]
Yes, check the wiki:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Web-services-description#motion-events-management

**Reply by Al3eRtO** [reply]
Thanks!Done, everything ok
